docs/config/README.md

```diff
@@ -21,10 +23,18 @@ $ tree ~/.minio
 │   ├── CAs
 │   ├── private.key
 │   └── public.crt
-└── config.json
 ```
 
 ### Configuration Fields
+#### Editing Configuration Fields
+For releases after minio.RELEASE.2017-08-05T00-00-53Z, configuration changes can be done using [mc]((https://docs.minio.io/docs/minio-client-quickstart-guide))
```

Comment:
https://github.com/minio/mc/blob/master/docs/minio-admin-complete-guide.md perhaps this link is better.

Reply:
done.